modules/ROOT/pages/create-partner.adoc

```diff
@@ -0,0 +1,73 @@
+= Create a New Partner
+
+Create a new partner by using the *Partners* page to specify the partner's name, identifiers, endpoints, validation rules, and other settings.
+
+== Prerequisites
+
+Before you create your first partner, xref:setup-payload-storage-API.adoc[make sure that you have configured your payload storage API].
+
+Obtain the following information:
+
+* The partner's name (usually the name of the company)
+* One or more partner identifiers (such as the AS2 ID, X12-ISA, X12 GS, and so on)
+
+== Create the Partner
+
+After meeting the prerequisites to create a partner:
+
+. In Partner Manager, select *Partners* in the navigation menu.
+. Click *Create partner > Partner*.
+. Enter the following information:
++
+[%header%autowidth.spread]
+|===
+| Setting a| Description
+| *Name*
+| Text string that easily identifies your partner
+| *Type*
+| Identifier type
+| *Qualifier*
+| Value that appears if you selected *X12-ISA* or *EDIFACT-UNB* as the type
+| *Identifier Value*
+| Value that uniquely identifies your partner
+|===
++
+. Click *Create partner*.
+
+== Configure Additional Partner Settings
+
+Add the following settings for the new partner, either now or later on:
+
+[%header%autowidth.spread]
+|===
+|Settings |Action a|Additional Information
+|Information about the partner, such as the partner's address, website, and contact details.
+| Click *Edit* in the *About* section.
+|
+| Additional identifiers to associate with the partner.
+| Click *New* in the *Identifiers* section.
+| xref:partner-manager-identifiers.adoc[Sender and Receiver Identifiers]
+| The partner's digital certificates, such as X.509 certificates.
+| Click *New* in the *Certificates* section.
+|
+| The partner-specific endpoints to use for sending and receiving messages to and from your partner, including acknowledgement endpoints. If you plan to use host endpoints only, you don't need to create partner-specific endpoints.
```

Comment:
```suggestion
| The partner-specific endpoints to use for sending and receiving messages to and from your partner, including acknowledgment endpoints. If you plan to use host endpoints only, you don't need to create partner-specific endpoints.
```

Reply:
You are correct that this is a British spelling, but that is how the GUI has it, so I am leaving it at least for now. (I will ask Kim about this next week.)